.eslintrc

```diff
@@ -115,6 +115,13 @@
       "rules": {
         "oppia/no-testonly": "off"
       }
+    },
+    {
+      "files": ["*.js", "*.ts"],
+      "excludedFiles": ["UpgradedServices.ts"],
```

Comment:
Do not exclude file, please add a disallow flag instead*

Reply:
Done!